js/index.js

```diff
@@ -22,6 +22,57 @@ for (let i = 0; i < skills.length; i++) {
     skill.innerHTML = skills[i];
     skillsList.appendChild(skill);
 }
+// 4.3 Form Basics
+//*******************************************************************************************
+// create messages
+// *******************************************************************************************
+// show the information from the form
+//--------------------------------------------------------------------------------------------
+const messageForm = document.getElementsByName("leave_message");
+console.log(messageForm[0], "Show the data from the form ");
 
+messageForm[0].addEventListener("submit", function(event){
+    event.preventDefault();
 
+const theMesssageForm = event.target;
```

Comment:
```suggestion
    const theMesssageForm = event.target;
```
This line and all the lines below should be indented because we're inside the function.